### PR TITLE
Build arm64 images

### PR DESCRIPTION
**Pull request by jakzal**
Previous attempts: https://github.com/jakzal/phpqa/pull/303 https://github.com/jakzal/phpqa/pull/306 https://github.com/jakzal/phpqa/pull/318 were reverted in https://github.com/jakzal/phpqa/commit/8034af81a5a435c9d921a51d74bdffe6ebe3b583.

### CODE REVIEW

**Comment by jakzal** [comment]
Currently, `--load` cannot be used with multiple platforms specified in `--platform`.

**Comment by jakzal** [comment]
Build times are terrible 
image

**Comment by ciaranmcnulty** [comment]
Can you re-run the Action? It may benefit from the GHA cache and be reasonable

To speed it up in terms of wall time:

1. Self-host an ARM worker. this can be a Raspberry Pi, AWS graviton2 ...
2. Optimise the ARM builds to share assets from the AMD ones

To speed up local builds:

3. Bake everything in one job at the start (that pushes to ghcr) to enable cross-PHP and cross-flavour asset sharing, have a second job that's testing them in a matrix, and a third single job that's copying them to docker hub on success

**Comment by ciaranmcnulty** [comment]
> Currently, `--load` cannot be used with multiple platforms specified in `--platform`.

Yep this is a limitation of the docker image store; it can't handle multi-arch references. Buildkit can't load it if there's nowhere for it to be stored

**Comment by ciaranmcnulty** [comment]
I don't see a reason not to merge this BTW

**Comment by ciaranmcnulty** [comment]
If you're worried about speed I'll do a `bake`-based spike after #366 as I think we can build the same-php images (alpine/debian + arm+amd) in a single bake and share the `toolbox install` between them

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -92,8 +92,7 @@ jobs:
                 uses: docker/build-push-action@v2
                 with:
                     context: ${{ matrix.flavour }}
-                    load: true
-                    platforms: linux/amd64
+                    platforms: linux/amd64,linux/arm64
```
[the remark this answers is not in the document]
Comment (jakzal) [reply .github/workflows/build.yml:95]: I need to build both to push both to docker hub, right?

**Comment by jakzal** [comment]
Build times after #366 was merged.

**The first build**

image


**The second build**

image

**Comment by jakzal** [comment]
I triggered the build again on another day to trigger `toolbox install` layer cache invalidation:

image

Here's a single arch build for comparison:

image

**Comment by ciaranmcnulty** [comment]
Looks good to me

**Pull request merged**